packages/pds/src/account-manager/index.ts

```diff
@@ -211,15 +211,21 @@ export class AccountManager
   async createSession(
     did: string,
     appPassword: password.AppPassDescript | null,
+    isSoftDeleted = false,
   ) {
     const { accessJwt, refreshJwt } = await auth.createTokens({
       did,
       jwtKey: this.jwtKey,
       serviceDid: this.serviceDid,
-      scope: auth.formatScope(appPassword),
+      scope: isSoftDeleted
```

Comment:
For a second, I thought we needed to replicate this logic on `rotateRefreshToken`, but `refreshSession` itself prevents session refresh when you're account is takendown.

I actually like that & don't think you should/need to be refreshing your session when taken down. Just making sure my understanding of this matches yours

Reply:
yep, we are actively making sure that refresh tokens for taken down accounts are not stored and as you said, we already fail attempts to rotate refreshtoken for takendown accounts so it can just be left alone.